auditlog_ext_nrq/models/auditlog_log_line.py

```diff
@@ -13,21 +14,26 @@ def create(self, vals):
         if vals.get('new_value_text') or vals.get('old_value_text'):
             old_value_text, new_value_text = vals.get(
                 'old_value_text'), vals.get('new_value_text')
+            timezone = pytz.timezone('Japan')
```

Comment:
@YogeshMahera-SerpentCS I suggested that we take the timezone from the superuser. We do not want to fix Japan in the code.

Reply:
Sorry @yostashiro, this is my mistake, Yogesh asked me on this yesterday and I told him we should fix the Japan timezone. Sorry for the miscommunication.

Reply:
Hi, @yostashiro @TimLai125 Amended changes with superuser timezone.:)